server.js

```diff
@@ -0,0 +1,100 @@
+const express = require("express");
+const ramenData = require("./ramenData")
+const path = require('path')
+
+
+const app = express();
+
+// middleware for css stylesheet
+const staticMiddleware = express.static(path.join(__dirname, 'public'))
+
+app.use(staticMiddleware)
+
+//logging middleware
+const morgan = require("morgan");
+
+app.use(morgan('dev'));
+
+//pulls ramenData
+const noodles = ramenData.list()
+
+app.get("/", (req, res, next) => {
```

Comment:
i think this route should be called `'/noodles'` to be consistent with the route on line 49
if you do this, you'd need a route like 
```js
app.get('/', (req, res, next) => res.redirect('/noodles'));
``` 
to make sure you never go to `'/'`